tests/test_smoke.py

```diff
@@ -2214,6 +2214,20 @@ def tmp_local_list_storage_obj(self, tmp_bucket_name, tmp_source):
         yield from self.yield_storage_object(name=tmp_bucket_name,
                                              source=list_source)
 
+    @pytest.fixture
+    def tmp_bulk_del_storage_obj(self, tmp_bucket_name, tmp_source):
```

Comment:
```suggestion
    def tmp_bulk_del_storage_obj(self, tmp_bucket_name):
```
`tmp_source` is unused in this fixture.